src/BridgedRoom.ts

```diff
@@ -345,6 +345,11 @@ export class BridgedRoom {
             username: user.getDisplaynameForRoom(message.room_id) || matrixToSlackResult.username,
         };
 
+        if (!body.attachments && !body.text) {
```

Comment:
What is attachments?!

Reply:
dunno, something we use in substitutions i guess :p